src/state/routing/slice.ts

```diff
@@ -1,15 +1,14 @@
 import { createApi, fetchBaseQuery, FetchBaseQueryError } from '@reduxjs/toolkit/query/react'
 import { Protocol } from '@uniswap/router-sdk'
 import { TradeType } from '@uniswap/sdk-core'
-import { AlphaRouter, ChainId } from '@uniswap/smart-order-router'
-import { RPC_PROVIDERS } from 'constants/providers'
-import { getClientSideQuote, toSupportedChainId } from 'lib/hooks/routing/clientSideSmartOrderRouter'
+import { ChainId } from '@uniswap/smart-order-router'
```

Comment:
random but do we export this from sdk-core as well? should we import from there?

Reply:
so interestingly this one should be from SOR specifically since these are chainIds that are accepted by the client side router - theres a chance that sdk-core will support a superset of these chainIds